frontend/src/metabase/dashboard/components/DashCard.styled.tsx

```diff
@@ -15,4 +16,10 @@ export const DashCardRoot = styled.div<DashCardRootProps>`
       border-color: ${color("bg-night")};
       background-color: ${color("bg-night")};
     `}
+
+  ${({ isUsuallySlow }) =>
+    isUsuallySlow &&
+    css`
```

Comment:
Just curious, why do we need `css` here? I'd guess that's because using `css` would make editors highlight style within the `css` tag rather than color them as a template string.